docs/deployment/.pages

```diff
@@ -1,7 +1,10 @@
 nav:
-  - README.md
-  - infrastructure.md
-  - secrets.md
-  - release.md
-  - troubleshooting.md
-  - workflows.md
+
+- README.md
+- infrastructure.md
+- secrets.md
+- release.md
+- hotfix.md
+- rollback.md
+- troubleshooting.md
+- workflows.md
```

Comment:
I think we want this to be an indented list still right?

I've noticed these `.pages` files sometimes revert to `markdown` formatting, when really they are closer to `yaml` (or even just plaintext).

Reply:
Yep, looks like the formatter made this change when I saved the file in the dev container. I redid the indentation and used `Save without Formatting` to bypass the automatic formatting because it insisted on reformatting this 😅 .

Reply:
I think it is because VS Code reverts to thinking this is a `markdown` file:

![image](https://github.com/user-attachments/assets/45ea3aae-d280-4464-a713-30a15c901546)

When really it is a `yaml` file (you can change this setting yourself by clicking on the language):

![image](https://github.com/user-attachments/assets/e61b4d9a-8690-4a83-826e-61b653584985)

We should look at configuring the file association in `.vscode/settings.json` (can be another PR).